<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1384</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <item>
     <widget class="QLabel" name="bigheading">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Information gain lēmumu koks</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="showoffabel">
      <property name="text">
       <string>veidojis Armands Vagalis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="csvButton">
         <property name="text">
          <string>Pievienot .csv</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="excelButton">
         <property name="text">
          <string>Pievienot Excel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ptlButton">
         <property name="text">
          <string>Pievienot .ptl</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutData">
      <item>
       <widget class="QScrollArea" name="optionbox">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <layout class="QVBoxLayout" name="verticalLayoutOption">
          <item>
           <widget class="QLabel" name="bigheading_2">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Datora loģika, veidojot koku:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="logicText">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="buttonBottomFrame">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="savevisButton">
               <property name="text">
                <string>Saglapāt koku .png</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="savevisSVGButton">
               <property name="text">
                <string>Saglapāt koku .svg</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mermaidButton">
               <property name="text">
                <string>Kopēt mermaid</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveptlButton">
               <property name="text">
                <string>Saglabāt .ptl</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="dataTableScroll">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <layout class="QVBoxLayout" name="verticalLayoutDataTable">
          <item>
           <widget class="QTableWidget" name="dataTable">
            <property name="rowCount">
             <number>5</number>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
